functions/nvm.fish

```diff
@@ -18,17 +18,20 @@ function nvm -a cmd -d "Node version manager"
         case -v --version
             echo "nvm, version $nvm_version"
         case "" -h --help
-            echo "usage: nvm install <version>  Download and activate a version"
-            echo "       nvm install            Install version from nearest .nvmrc file"
-            echo "       nvm use <version>      Activate a version in the current shell"
-            echo "       nvm use                Activate version from nearest .nvmrc file"
-            echo "       nvm remove <version>   Remove an installed version"
-            echo "       nvm current            Print currently-active version"
-            echo "       nvm list               List installed versions"
-            echo "       nvm list-remote        List versions available to install"
+            echo "usage: nvm install <version>    Download and activate a version"
+            echo "       nvm install              Install version from nearest .nvmrc file"
+            echo "       nvm use <version>        Activate a version in the current shell"
+            echo "       nvm use                  Activate version from nearest .nvmrc file"
+            echo "       nvm uninstall <version>  Remove an installed version"
```

Comment:
Should description also say "uninstall" here?

Reply:
Whoops!